README.md

```diff
@@ -0,0 +1,22 @@
+# Sylt-lang
+-----------
+Sylt is a statically-checked dynamically typed, reference counted programming
+language made for GameJams.
+
+
+## Why does this exist? Why not use X-lang?
+Pfft! Why not?
+    
+
+## Getting started
+Sylt is written entirely in Rust. Pointing to this Git-repo 
+in your Cargo.toml is the easiet way. Alternatively you can
+build the interpreter and run your own programs from there.
+
+
+## Endgame
```

Comment:
```suggestion
## Long-term goals
```

Reply:
Don't agree, we'll have to discuss this.